doc/plugins/udapp.md

```diff
@@ -8,7 +8,7 @@ The udapp exposes an interface for interacting with the account and transaction.
 |Type     |Name                 |Description
 |---------|---------------------|--
 |_event_  |`newTransaction`     |Triggered when a new transaction has been sent.
-|_method_ |`sendTransaction`    |Send a transaction **only for testing networks**.
+|_method_ |`sendTransaction`    |Send a transaction **only for JavaScript VM**.
```

Comment:
We'll update the remix-ide to support testing networks. Could you rollback this change ?
Same below.

Reply:
Updated so it only includes the solidity -> udapp change (and i guess a newline character at the end of the file that github added on its own)